charts/alluxio/templates/master/statefulset.yaml

```diff
@@ -42,6 +42,8 @@ metadata:
     uid: {{ .Values.owner.uid }}
   {{- end }}
 spec:
+  updateStrategy:
```

Comment:
I suggest defining the updateStrategy in values.yaml so it can be easily configured later.